doc/psud/PSU_daemon_design.md

```diff
@@ -4,10 +4,13 @@
 
 ### Revision ###
 
+
  | Rev |     Date    |       Author       | Change Description                |
  |:---:|:-----------:|:------------------:|-----------------------------------|
  | 0.1 |             |      Chen Junchao  | Initial version                   |
  | 0.2 | August 4st, 2022 | Stephen Sun | Update according to the current implementation |
+ | 0.3 | August 8st, 2022 | Or Farfara | Add input current, voltage and max power |
```

Comment:
`8th`. typo. can you also fix the typo `4st` in line 11? thank you.

Reply:
Sure,done